### ISSUE TITLE
JCPNEXT4-23: Problems when non-member companies encourage their employees to join as individuals

### ISSUE THREAD

**Issue by apastsya** [jcp-org/jcpnext4#6]
Jira issue originally created by user pcurran:

In order to avoid paying membership fees companies sometimes do not join the JCP themselves but instead encourage one or more of their employees to join as individuals, typically in order to join an Expert Group.

Under these circumstances the employee may well be acting as an "Agent" of the employer, and may (deliberately or inadvertently) contribute IP belonging to the employer, but the employer makes absolutely no IP grants. (Exhibit B simply asserts that the employer releases the employee of any claims over "personal" IP that the employee may otherwise have owned.)

In other words, the protection provided by the Royalty-Free grants in respect to Contributions to a particular JSR and the RAND grants in respect to essential patents for other JSRs are completely absent.

Specifically:

1) The work of an Expert Group that the employee is participating in may become "contaminated" by IP belonging to the employer (without any legal grant of IP rights.)

2) The employer avoids the obligation to make RAND grants with respect to essential patents for JSRs that the employee does not participate in.
